meta-lmp-bsp/recipes-bsp/u-boot/u-boot-ostree-scr-fit/uz/boot.cmd

```diff
@@ -21,6 +21,9 @@ setenv bootloader2_image "u-boot.itb"
 setenv bootloader2_s_image ${bootloader2_image}
 
 if test "${modeboot}" = "qspiboot"; then
+	# Use SD for further loading/booting Linux FIT image
+	setenv devnum 1
```

Comment:
Is eMMC 0 here? Just because we will also have to support eMMC, just need to find out what to do (if continue with sdcard by default, if switch to emmc or if try both).

Reply:
I remember you mentioned that we should boot from eMMC, but for testing purposes I've added SD. I can change to eMMC

Reply:
Can we have sdcard for open devices and emmc for closed ones? That might work better as we expect to use fiovb on closed devices.

Reply:
@ricardosalveti done, also added bump commit for u-boot-xlnx